### ISSUE TITLE
Design database schema

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#33]
## User story
1. As a {Backend Develpoper/ Data Architect}, I want to design a comprehensive database schema that efficiently organizes and stores geospatial data so that it can be easily accessed, queried, and analyzed by our application.
2. I want / need {to ensure the schema accommodates various data types and sources, including spatial data types and traditional relational data.}
3. So that {the backend system can deliver high-performance data retrieval and provide a robust platform for future scalability and data integration.}

## Acceptance criteria
* The schema supports geospatial data types and is optimized for spatial queries.
* The design includes normalization to reduce redundancy and improve data integrity without sacrificing query performance.
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
